### ISSUE TITLE
Use https endpoint

### ISSUE THREAD

**Issue by pdevine** [timehop/go-mixpanel#13]
Hey peeps,

It'd be great to be able to use the https endpoint instead of the http endpoint for mixpanel.  Apparently both work fine in the testing that I've been doing.  From my read of the code, it looks like the API token is base64 encoded, but nothing else ends up protecting it over port 80, so it'd be possible to extract with with something like wireshark.

**Comment by burdiyan** [comment]
I saw Mixpanel Python library uses https endpoint so this one should be used in production. Is this library still maintained? No activity for a long time.

**Comment by pdevine** [comment]
yeah..  seems pretty dead.  I forked and changed the endpoint here:  https://github.com/pdevine/go-mixpanel
